packages/amazonq/.changes/next-release/Bug Fix-6f801ee1-63c1-4688-aa74-40a67d3a9193.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "fix(featureDev): stop showing spinerText when there is an error."
```

looks like a typo/spelling error. Note also the primary audience for this is customers. Please review https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog .

```suggestion
	"description": "Amazon Q /dev command: stop showing spinner when there is an error."
```

---

Make sense. Going to commit your suggestion.